app/build.gradle

```diff
@@ -87,6 +87,7 @@ dependencies {
     implementation 'org.mousio:etcd4j:2.11.0'
     implementation 'com.googlecode.json-simple:json-simple:1.1.1'
     implementation 'org.apache.logging.log4j:log4j-core:2.14.1'
+    implementation 'org.webjars.bowergithub.cloudogu:ces-theme:0.2.1'
```

Comment:
Today I saw that a new version was released? We should consider upgrading as well.